start.sh

```diff
@@ -1,16 +1,20 @@
 #!/usr/bin/env bash
 
 if [[ "$TAUTULLI_DOCKER" == "True" ]]; then
-    PUID=${PUID:-1000}
-    PGID=${PGID:-1000}
+    if [[ "$TAUTULLI_DOCKER_GOSU" == "True" ]]; then
+        PUID=${PUID:-1000}
+        PGID=${PGID:-1000}
 
-    groupmod -o -g "$PGID" tautulli
-    usermod -o -u "$PUID" tautulli
+        groupmod -o -g "$PGID" tautulli
+        usermod -o -u "$PUID" tautulli
 
-    find /config \! \( -uid $(id -u tautulli) -gid $(id -g tautulli) \) -print0 | xargs -0r chown tautulli:tautulli
+        find /config \! \( -uid $(id -u tautulli) -gid $(id -g tautulli) \) -print0 | xargs -0r chown tautulli:tautulli
 
-    echo "Running Tautulli using user tautulli (uid=$(id -u tautulli)) and group tautulli (gid=$(id -g tautulli))"
-    exec gosu tautulli "$@"
+        echo "Running Tautulli using user tautulli (uid=$(id -u tautulli)) and group tautulli (gid=$(id -g tautulli))"
+        exec gosu tautulli "$@"
+    else
+        exec tautulli "$@"
```

Comment:
Don't you still need to create a `tautulli` user?

Reply:
That doesn't matter, using kubernetes or docker compose people can override the user and group anyways as long as they have their permissions on the volume set correctly.

Reply:
I meant that this command will fail because there is no `tautulli` user inside the container.

Reply:
Actually I see what you mean the tautulli in the command is the user.

Reply:
Updated.